site/src/main/kotlin/react/bootstrap/site/components/docs/components/alerts/Helpers.kt

```diff
@@ -8,19 +8,20 @@ import react.bootstrap.components.alert.closingElement
 import react.bootstrap.components.alert.dismissibleAlert
 import react.bootstrap.components.alert.heading
 import react.bootstrap.components.alert.link
+import react.bootstrap.site.components.docs.buildNestedName
 import kotlin.reflect.KClass
 
 internal val KClass<out Alert.Variants>.nestedName: String
-    get() = "${Alert::class.simpleName}.${this.simpleName}"
+    get() = buildNestedName(this::class, Alert::class)
 
 internal val KClass<out Alert.Props>.nestedName: String
-    get() = "${Alert::class.simpleName}.${this.simpleName}"
+    get() = buildNestedName(this::class, Alert::class)
 
 internal val KClass<out Alert.DismissibleProps>.nestedName: String
-    get() = "${Alert::class.simpleName}.${this.simpleName}"
+    get() = buildNestedName(this::class, Alert::class)
 
 internal val Alert.Variants.kt: String
-    get() = "${this::class.nestedName}.$name"
+    get() = buildNestedName(this::class, Alert::class)
```

Comment:
nope! this will print Alert.Variants
```suggestion
    get() = buildNestedName(name, Alert::class, Variants::class)
```